shell/renderer/electron_sandboxed_renderer_client.cc

```diff
@@ -90,6 +90,12 @@ v8::Local<v8::Value> CreatePreloadScript(v8::Isolate* isolate,
                                        preloadSrc);
 }
 
+double Uptime() {
+  return (base::Time::NowFromSystemTime() -
```

Comment:
What's the difference between `base::Time::Now` and `base::Time::NowFromSystemTime`? In what situations will these return different values?

Reply:
From a bit of searching, it looks like the difference is only (a) during tests and (b) when "virtual time" is enabled, either through `--initial-virtual-time` or the [Emulation.setVirtualTimePolicy](https://chromedevtools.github.io/devtools-protocol/tot/Emulation/#method-setVirtualTimePolicy) devtools protocol method is called.

It seems to me like there's not any particular reason to bypass the override in those cases, so perhaps `base::Time::Now` is the better choice here?

Reply:
changed to `base::Time::Now`